app/controllers/glossary_terms_controller.rb

```diff
@@ -26,10 +26,8 @@ def show
   # ---------- Actions to Display forms -- (new, edit, etc.) -------------------
 
   def new
-    @copyright_holder = @user.name
-    @copyright_year = Time.now.utc.year
-    @upload_license_id = @user.license_id
-    @licenses = License.current_names_and_ids(@user.license)
+    @glossary_term = GlossaryTerm.new
+    init_image_form_instance_variables
```

Comment:
Do you really mean "image" here and not "term"?

Reply:
Oh, sorry, just confused, ignore me!

Reply:
Yep. It's real confusing. I wish I could have found a better name.